packages/table/src/headers/columnHeaderCell.tsx

```diff
@@ -128,7 +139,7 @@ export class ColumnHeaderCell extends AbstractComponent<IColumnHeaderCellProps,
     }
 
     protected validateProps(nextProps: IColumnHeaderCellProps) {
-        if (nextProps.menu != null) {
+        if (nextProps != null && nextProps.menu != null) {
```

Comment:
It was undefined (with console error) and this check fixed it. I'll investigate more tomorrow and get back to you